### ISSUE TITLE
[Suggestion] Rail speed control (part 99)

### ISSUE THREAD

**Issue by ZachZent** [tsunamayo/Starship-EVO#2591]
Now that we got pistons with speed control, any chance we can get it with sliders?

**Comment by tsunamayo** [comment]
yeah I will work on that, problem will be with rotator, I want to get rid of the axis assignment first... Thanks

**Comment by Xerma** [comment]
Being able to control slider speeds would be amazing. At the moment the only option to make an elevator that isn't slower than taking the stairs is to use multiple pistons which I'd rather keep my entity count as low as possible
